app/Agri.Models/Configuration/Feed.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Agri.Models.Configuration
+{
+    public class Feed : Versionable
+    {
+        public Feed()
+        {
+            FeedForageTypes = new List<FeedForageType>();
+        }
+
+        [Key]
+        public int Id { get; set; }
+
```

Comment:
the foreign key should be here to FeedForage Type:
        public int FeedForageTypeId { get; set; }